src/DenseACSets.jl

```diff
@@ -644,6 +644,26 @@ end
 ACSetInterface.cascading_rem_parts!(acs::ACSet, type, parts) =
   delete_subobj!(acs, Dict(type=>parts))
 
+function ACSetInterface.undefined_subparts(acs::StructACSet{S,Ts}, f::Symbol) where {S,Ts}
+  s = Schema(S)
+  f ∈ homs(s; just_names=true) || error("$f is not a hom")
+  _undefined_subparts(acs, f, acs.parts[dom(s,f)])
+end
+
+function ACSetInterface.undefined_subparts(acs::DynamicACSet, f::Symbol)
+  s = Schema(acs.schema)
+  f ∈ homs(s; just_names=true) || error("$f is not a hom")
+  _undefined_subparts(acs, f, acs.parts[dom(s,f)])
+end
+
+function _undefined_subparts(acs::SimpleACSet, f::Symbol, ::DenseParts)
+  findall([!haskey(acs.subparts[f],i) for i in dom_parts(acs,f)])
+end
+
+function _undefined_subparts(acs::SimpleACSet, f::Symbol, ::MarkAsDeleted)
+  error("undefined_subparts only defined for homs with DenseParts domains")
```

Comment:
Would it make sense for `undefined_subparts` to not error here but instead return the (undeleted) parts of the domain which are pointing at a part that has been marked as deleted in the codomain?

Reply:
Good point! Yes, I'll implement that.